### PR TITLE
Improve README files for examples

### PR DESCRIPTION
**Pull request by llawall**
Add some readme files for all examples so they render when entering each time the example directory is opened in GitHub.

### CODE REVIEW

File: examples/binning/live-plot.ipynb [hunk 1]
```diff
@@ -13,12 +13,16 @@
...
+    "\n",
     "Once installed, run these two commands in examples/binning/ :\n",
     "\n",
     " * The binning stream processor:\n",
     "   * python binning.py\n",
     " * The price generator:\n",
-    "   * python generator.py     -i A.N.Other-Corp,0.3,123,100.0,0.01     -l 6000 -f 250ms     -kb localhost:9092 -kt prices     -rt\n"
+    "   * python generator.py     -i A.N.Other-Corp,0.3,123,100.0,0.01     -l 6000 -f 250ms     -kb localhost:9092 -kt prices     -rt\n",
+    "\n",
+    "The -rt argument will generate the prices in real-time. The prices can also be generated faster than realitime with the flag "-rtm <multiplier>" or in bulk by omitting the -rt flag"
```
Comment (ah-) [inline examples/binning/live-plot.ipynb]: Is this just github being weird or is the quoting funny?

File: examples/wordcount/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Debug Winton Kafka Streams Example
```
Comment (ah-) [inline examples/wordcount/README.md]: Wordcount?

File: examples/wordcount/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+## Running
+* Edit the config.properties file if necessary to change where Kafka is running
+* Run: python example.py
+* Start a console producer writing to the topic 'wks-debug-example-topic-two'
```
Comment (ah-) [inline examples/wordcount/README.md]: wordcount?

File: examples/wordcount/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+## Features
+* Listens to the topic 'wks-wordcount-example-topic' and writes output to 'wks-wordcount-example-count'
+* Each string read in will be split by spaces
+* The count of the number of words will be maintained in a collections.Counter instance. This is not persistent so stopping the exmaple will not maintain the previous state.
```
Comment (ah-) [inline examples/wordcount/README.md]: example?

File: examples/binning/live-plot.ipynb (hunk 1 above)
Comment (llawall) [inline examples/binning/live-plot.ipynb]: No, this is my mistake

**Pull request merged**